pulumi/folio/kafka.ts

```diff
@@ -0,0 +1,25 @@
+import * as k8s from "@pulumi/kubernetes";
+
+// Deploy Kafka using the Helm chart.
+export module kafka {
+    export function Helm(kubeconfig: any, appsNamespaceName: string) {
```

Comment:
Another thing that might be more idiomatic here would be to put a verb in the function name to indicate what it is doing, so something like `deployKafka` or just `deploy` since kafka is already there in the module name. How it is doing its work (via helm) might be considered an implementation detail. In OO land it's good practice to conceal (encapuslate) how the work is done inside the class/module. Also usually in JS functions are camel cased starting with a lowercase letter. Classes (things that get "newed") are usually given uppercase names, so you'd get something like:

```
var cat = new Cat()
cat.meow()
```

So in this case making the function name start with an uppercase letter might lead someone to believe it is something that can be newed up, which, because it is a javascript function, [it can be](https://stackoverflow.com/questions/1646698/what-is-the-new-keyword-in-javascript). Classes are a newer JS construct, and while they get newed, functions can be too.

Reply:
What do you think about switching to something like this? Get's rid of the `kafka.kafka` and the case issues, doesn't encapsulate, but I kind of like the added information when calling the function. It could make it easy to deprecate later if we end switching to MSK (replace `kafka.deploy.helm` with `kafka.deploy.awsMsk`).
```ts
// Deploy Kafka using the Helm chart.
export module deploy {
    export function helm(kubeconfig: any, appsNamespaceName: string) {
```

Reply:
LGTM - I like the semantics of `<thing>.deploy.<the method of deployment>`.